conans/test/functional/cache/download_cache_test.py

```diff
@@ -122,7 +122,7 @@ def source(self):
            """ % http_server.port)
         client.save({"conanfile.py": conanfile})
         client.run("source .", assert_error=True)
-        self.assertIn("ConanException: md5 signature failed", client.out)
+        self.assertIn("ConanException: All 1 URLs have failed: md5 signature failed", client.out)
```

Comment:
I don't like having this error message when you only provide 1 url, it makes the output a bit more confusing to the user.

Reply:
I see, I'll filter it raise a specific message when there is only one URL.